.changeset/afraid-crabs-ring.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `selector-type-*`, `selector-max-type` and `selector-class-pattern` false positives for `<timeline-range-name>` in keyframe selectors
```

Comment:
question
minor

Does it underline the percentage too?
i.e. Should it be "…for `<timeline-range-name> <percentage>` in…"?

Reply:
I guess it depends on the rule?
I think the current text is descriptive enough to communicate what was fixed.